app/browser/api/ledger.js

```diff
@@ -2212,7 +2226,7 @@ const migration = (state) => {
         }
       })
     } catch (err) {
-      console.log('Error migrating file', err.toString())
+      console.log(err.toString())
```

Comment:
nit console.error here so it has callstack too

Reply:
will check the whole `ledger.js` file and replace it where needed

Reply:
done